src/main/java/training/commands/Command.java

```diff
@@ -1,4 +1,4 @@
-package training.supportbank;
+package training.commands;
 
 public interface Command {
```

Comment:
This could be an abstract class rather than an interface (I think I suggested this code originally - oops!).